src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { LinkArrowIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  label: string;
+  href: string;
+  size?: 'small' | 'medium' | 'large';
+  outlined?: boolean;
+  rounded?: boolean;
+  arrow?: boolean;
```

Comment:
I suggest to rename it to `icon?: string`.

Reply:
Renamed and added icon selection logic